demo/Demo/Textfields.elm

```diff
@@ -224,6 +225,32 @@ textfields model =
         ]
        """
     )
+  , ( "Expandable textfield"
+    , Textfield.render Mdl [11] model.mdl
+        [ Textfield.label "Expandable"
+        , Textfield.floatingLabel
+        -- NOTE: The id's must match
```

Comment:
Unfortunately I don't think it's possible to easily detect setting the attributes due to `Options.id` and `Options.attribute <| Html.Attributes.id` get turned into `Html.Attribute` and we cannot access those values once set.